activerecord/test/cases/finder_test.rb

```diff
@@ -1180,6 +1180,13 @@ def test_find_with_order_on_included_associations_with_construct_finder_sql_for_
       order("author_addresses_authors.id DESC").limit(3).to_a.size
   end
 
+  def test_find_with_eager_loading_collection_and_ordering_by_collection_primary_key
+    Comment.delete_all
+    Post.delete_all
```

Comment:
`delete_all`s are unnecessary.

Reply:
If removing `delete_all`s, the test succeeds even without applying the patch, doesn't it?

Reply:
I see. Prefer reusing existing fixture like `assert_equal Post.first, Post.eager_load(comments: :ratings).order("posts.id, ratings.id, comments.id").first`.